jakarta-ee-messaging/src/main/kotlin/io/qalipsis/examples/jakarta/JakartaProduceAndConsumeFromTopic.kt

```diff
@@ -0,0 +1,128 @@
+package io.qalipsis.examples.jakarta
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import io.kotest.assertions.asClue
+import io.kotest.assertions.assertSoftly
+import io.kotest.matchers.ints.shouldBeExactly
+import io.qalipsis.api.annotations.Scenario
+import io.qalipsis.api.executionprofile.regular
+import io.qalipsis.api.scenario.scenario
+import io.qalipsis.api.steps.filterNotNull
+import io.qalipsis.api.steps.innerJoin
+import io.qalipsis.api.steps.map
+import io.qalipsis.api.steps.verify
+import io.qalipsis.examples.utils.BatteryState
+import io.qalipsis.examples.utils.ScenarioConfiguration
+import io.qalipsis.examples.utils.ServerConfiguration
+import io.qalipsis.plugins.jackson.csv.csvToObject
+import io.qalipsis.plugins.jackson.jackson
+import io.qalipsis.plugins.jakarta.consumer.consume
+import io.qalipsis.plugins.jakarta.deserializer.JakartaJsonDeserializer
+import io.qalipsis.plugins.jakarta.destination.Topic
+import io.qalipsis.plugins.jakarta.jakarta
+import io.qalipsis.plugins.jakarta.producer.JakartaMessageType
+import io.qalipsis.plugins.jakarta.producer.JakartaProducerRecord
+import io.qalipsis.plugins.jakarta.producer.produce
+import jakarta.jms.Session
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory
+
+class JakartaProduceAndConsumeFromTopic {
+
+    private val objectMapper = ObjectMapper().also {
+        it.registerModule(JavaTimeModule())
+    }
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    @Scenario("jakarta-produce-and-consume-from-topic")
+    fun scenarioProduceAndConsumeFromTopic() {
+
+        val factory = ActiveMQConnectionFactory(
```

Comment:
Please do not use variable, and do not reuse the factory, connection and session. This leads to misunderstanding from the readers.